x/ibc/perm/types/codec.go

```diff
@@ -11,13 +11,13 @@ import (
 // RegisterLegacyAminoCodec registers the necessary x/ibc perm interfaces and concrete types
 // on the provided LegacyAmino codec. These types are used for Amino JSON serialization.
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	legacy.RegisterAminoMsg(cdc, &MsgSetPermissionedRelayer{}, "ibc-perm/MsgSetPermissionedRelayer")
+	legacy.RegisterAminoMsg(cdc, &MsgSetPermissionedRelayers{}, "ibc-perm/MsgSetPermissionedRelayer")
```

Comment:
need to change `ibc-perm/MsgSetPermissionedRelayer` to `ibc-perm/MsgSetPermissionedRelayers`

Reply:
still the codec value is `ibc-perm/MsgSetPermissionedRelayer` need to add s